### ISSUE TITLE
Licensing issues

### ISSUE THREAD

**Issue by odinserj** [agladky/Comquer#1]
This project is heavily based on the [ModelFramework](https://github.com/ChessOK/ModelFramework/tree/master/ModelFramework) project that is licensed under the [MIT license](https://raw.githubusercontent.com/ChessOK/ModelFramework/master/LICENSE.txt), &copy;&nbsp;Convekta Ltd, 2011. The whole project structure, code and comments are almost the same. ModelFramework's license text says the following:

> Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the **rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software**, and to permit persons to whom the Software is furnished to do so, **subject to the following conditions**:
>
> **The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.**
> ...

So in other words, you can copy and rename the original project, but you have to include the original copyright notice into the repository. Otherwise this project is just a speculation on open source software and community.
